### ISSUE TITLE
Mail notify

### ISSUE THREAD

**Issue by nicotravassos** [hherzl/ServiceMonitor.NetCore#1]
Good day,

Love this app, could I ask if you can add ability to mail us based on the activity of the service for example if it has stopped or restarted etc.

Ofcause this feature can be set active by end user by activating it and entering a mail destination.

**Comment by hherzl** [comment]
Hi, thank you for your message.

With "service", do you mean monitor service or service is watching (database, web service) ?

Regards

**Comment by nicotravassos** [comment]
Preferably the service it is watching database and webservice just a time based mail what the service is up to.

Time based (hourly, daily..)

**Comment by hherzl** [comment]
Please let me know if I get your feedback.

Do you want a way to fire an action if a service if fails after of a success watch?

Regards

**Comment by nicotravassos** [comment]
Yes that is 100% correct! Also would it possible to watch windows services and notify if a service has stopped?

**Comment by hherzl** [comment]
Sorry for my delay reply.

I'll improve this tool, but at this moment I don't have time I have a lot of work...

Also I need to migrate this project to .NET Core 2

I'll let you know when I complete those tasks.

Regards

**Comment by nicotravassos** [comment]
Thank you @hherzl
